### PR TITLE
Dont stratify assay types by level, since level can be filtered in another menu

### PR DESCRIPTION
**Pull request by adamabeshouse**
Fixes https://github.com/ncihtan/htan-portal/issues/89

### CODE REVIEW

File: lib/helpers.ts [hunk 1]
```diff
@@ -168,4 +169,28 @@ export function sortStageOptions(options:ExploreOptionType[]){
...
+    const splitByLevel = t.split("Level");
+    const level = splitByLevel.length > 1 ? `Level ${splitByLevel[1]}` : null;
+    const extractedName = splitByLevel[0].match(/bts:(.+)/);
+    if (extractedName && extractedName[1]) {
+        // Convert camel case to space case
+        // Source: https://stackoverflow.com/a/15370765
+        let name = extractedName[1].replace( /([A-Z])([A-Z])([a-z])|([a-z])([A-Z])/g, '$1$4 $2$3$5' );
+
+        // Handle special case: scRNA
+        name = name.replace(/Sc RNA\b/g, "scRNA");
```
Comment (inodb) [inline lib/helpers.ts:189]: prolly anything prefixed with `Sc ` should be changed in this way 

e.g. `Sc ATAC-seq` should be `scATAC-seq`

Comment (inodb) [reply lib/helpers.ts:189]: sc is short for single cell

**Review by inodb — APPROVED** [review]
Thanks! LGTM - one minor comment about `Sc ATAC-Seq`

File: lib/helpers.ts (hunk 1 above)
Comment (inodb) [reply lib/helpers.ts:189]: same applies to `sn` (single nucleus), see data standards list here:

https://htan-portal-nextjs-ln46lfjfo.vercel.app/standards

Comment (adamabeshouse) [reply lib/helpers.ts:189]: okay cool, Ill just always treat a lone sc as a prefix

**Pull request merged**